pkg/driver/driver.go

```diff
@@ -46,6 +49,21 @@ const (
 	maxAttempts = 5
 )
 
+type draPlugin interface {
+	PublishResources(context.Context, resourceslice.DriverResources) error
+	Stop()
+	RegistrationStatus() *registerapi.RegistrationStatus
+}
+
+type resourceDB interface {
```

Comment:
We can keep this name like `inventoryDB`. Keeping it same as the original name is simple here (and we don't have a specific case where we want to name it differently)